srcpkgs/kdenlive/template

```diff
@@ -10,7 +10,8 @@ makedepends="
  kdeclarative-devel kfilemetadata5-devel knewstuff-devel knotifyconfig-devel
  kplotting-devel mlt7-devel qt5-multimedia-devel qt5-webkit-devel purpose-devel
  v4l-utils-devel ksolid-devel qt5-quickcontrols2-devel qt5-networkauth-devel"
-depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit qt5-quickcontrols"
+depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit qt5-quickcontrols
+ kirigami2 mediainfo"
```

Comment:
Appears `qt5-graphicaleffects` might be missing too as per:
```
$ kdenlive
...
qrc:/qml/timeline.qml:1177:25: Type TrackHead unavailable 
                             TrackHead { 
                             ^
qrc:/qml/TrackHead.qml:10:1: module "QtGraphicalEffects" is not installed 
     import QtGraphicalEffects 1.15 
     ^
QObject::connect: Cannot connect (nullptr)::mousePosChanged(int) to MainWindow::slotUpdateMousePosition(int)
QObject::connect: Cannot connect (nullptr)::zoomIn(bool) to MainWindow::slotZoomIn(bool)
QObject::connect: Cannot connect (nullptr)::zoomOut(bool) to MainWindow::slotZoomOut(bool)
QObject::connect: Cannot connect (nullptr)::processingDrag(bool) to MainWindow::enableUndo(bool)
Segmentation fault
```
(full launch log https://termbin.com/ex3i)

Reply:
hm i have that installed but I still have the segfault